### ISSUE TITLE
Initialize Quaternion from identity Matrix4 yields nan values

### ISSUE THREAD

**Issue by DJBen** [nicklockwood/VectorMath#12]
```
(lldb) po Quaternion(rotationMatrix: Matrix4.identity)
▿ Quaternion
  - x : nan
  - y : nan
  - z : 0.0
  - w : nan
```

**Comment by nicklockwood** [comment]
Thanks, I'll look into it.

**Comment by nicklockwood** [comment]
Fixed in 0.3.1
